### PR TITLE
use conda-forge instead of anaconda

### PR DESCRIPTION
**Pull request by nvnieuwk**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

**Review by maxulysse — APPROVED** [review]

File: modules/nf-core/homer/maketagdirectory/environment.yml [hunk 1]
```diff
@@ -9,4 +9,4 @@ dependencies:
   - conda-forge::r-base=4.0.2
   - bioconda::bioconductor-deseq2=1.30.0
   - bioconda::bioconductor-edger=3.32.0
-  - anaconda::perl=5.26.2
+  - conda-forge::perl=5.26.2
```
Comment (ewels) [inline modules/nf-core/homer/maketagdirectory/environment.yml:12]:
```suggestion
  - conda-forge::perl=5.32.1.1
```

File: modules/nf-core/gawk/environment.yml [hunk 2]
```diff
@@ -4,4 +4,4 @@ channels:
   - bioconda
   - defaults
 dependencies:
-  - anaconda::gawk=5.1.0
+  - conda-forge::gawk=5.1.0
```
Comment (ewels) [inline modules/nf-core/gawk/environment.yml:7]:
```suggestion
  - conda-forge::gawk=5.3.0
```

File: modules/nf-core/cadd/environment.yml [hunk 3]
```diff
@@ -5,5 +5,5 @@ channels:
   - defaults
 dependencies:
   - bioconda::cadd-scripts=1.6.post1
-  - anaconda::conda=4.14.0
+  - conda-forge::conda=4.14.0
   - conda-forge::mamba=1.4.0
```
Comment (ewels) [inline modules/nf-core/cadd/environment.yml:9]: Conda and mamba in one recipe? Why?

**Review by ewels — COMMENTED** [review]
May as well update the others I guess. Also conda + mamba??

File: modules/nf-core/cadd/environment.yml (hunk 3 above)
Comment (nvnieuwk) [reply modules/nf-core/cadd/environment.yml:9]: @ramprasadn can you answer this since you wrote the module?

**Pull request merged**